pypeit/spectrographs/keck_hires.py

```diff
@@ -59,7 +59,7 @@ class KECKHIRESSpectrograph(spectrograph.Spectrograph):
     url = 'https://www2.keck.hawaii.edu/inst/hires/'
     pypeline = 'Echelle'
     ech_fixed_format = False
-    supported = True
+    supported = False
```

Comment:
Should this be True now?

Reply:
I don't think yet, e.g. we haven't implemented anything
sensible for flat fielding.  Am going to add notes here
on what else needs to be done before making it supported